src/cargo/core/resolver/version_prefs.rs

```diff
@@ -59,7 +59,7 @@ impl VersionPreferences {
     ///
     /// Sort order:
     /// 1. Preferred packages
-    /// 2. [`VersionPreferences::max_rust_version`]
+    /// 2. Most compatible [`VersionPreferences::rust_versions`]
```

Comment:
The [unstable doc](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#msrv-aware-resolver) might need an update according, for the algorithm of determining workspace MSRV. However, from a user perspective, it is a bit hard to understand. I am not sure if we want to document it.

It is also hard to determine from a human eye, if it is a large workspace and people are adding/removing members a lot that changes the workspace-wide MSRV.

Reply:
I've updated the text.  The unstable docs offer us an opportunity to evaluate the wording.  Even once its stabilized, we can iterate as we see how people are dealing with it.